Gemfile.lock

```diff
@@ -1,75 +1,75 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actioncable (6.0.2.2)
-      actionpack (= 6.0.2.2)
+    actioncable (6.0.3.2)
+      actionpack (= 6.0.3.2)
       nio4r (~> 2.0)
       websocket-driver (>= 0.6.1)
-    actionmailbox (6.0.2.2)
-      actionpack (= 6.0.2.2)
-      activejob (= 6.0.2.2)
-      activerecord (= 6.0.2.2)
-      activestorage (= 6.0.2.2)
-      activesupport (= 6.0.2.2)
+    actionmailbox (6.0.3.2)
+      actionpack (= 6.0.3.2)
+      activejob (= 6.0.3.2)
+      activerecord (= 6.0.3.2)
+      activestorage (= 6.0.3.2)
+      activesupport (= 6.0.3.2)
       mail (>= 2.7.1)
-    actionmailer (6.0.2.2)
-      actionpack (= 6.0.2.2)
-      actionview (= 6.0.2.2)
-      activejob (= 6.0.2.2)
+    actionmailer (6.0.3.2)
+      actionpack (= 6.0.3.2)
+      actionview (= 6.0.3.2)
+      activejob (= 6.0.3.2)
       mail (~> 2.5, >= 2.5.4)
       rails-dom-testing (~> 2.0)
-    actionpack (6.0.2.2)
-      actionview (= 6.0.2.2)
-      activesupport (= 6.0.2.2)
+    actionpack (6.0.3.2)
+      actionview (= 6.0.3.2)
+      activesupport (= 6.0.3.2)
       rack (~> 2.0, >= 2.0.8)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.2.0)
-    actiontext (6.0.2.2)
-      actionpack (= 6.0.2.2)
-      activerecord (= 6.0.2.2)
-      activestorage (= 6.0.2.2)
-      activesupport (= 6.0.2.2)
+    actiontext (6.0.3.2)
```

Comment:
### [actiontext](https://rubyonrails.org)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 6.0.2.2 → 6.0.3.2

[_[change-log](https://github.com/rails/rails/blob/v6.0.3.2/actiontext/CHANGELOG.md), [source-code](https://github.com/rails/rails/tree/v6.0.3.2/actiontext)_]





<details>
<summary>Commits</summary>

A change of **260** commits. See the full changes on [the compare page](https://github.com/rails/rails/compare/v6.0.2.2...v6.0.3.2).

These are the first 10 commits:
- (dc84717) [Add file analysis to the Active Storage guide [ci skip]](https://github.com/rails/rails/commit/dc847176647a3f377ef4d3065921b82ca8019eea)
- (675f967) [Controller can be symbols as well](https://github.com/rails/rails/commit/675f967a314c3029137d1ed6a306c85d5a4a958f)
- (480745c) [Merge pull request #37862 from giraffate/fix_random_ci_failu…](https://github.com/rails/rails/commit/480745c56752fb381ba0b84b59eb58cec62eb72e)
- (596e125) [Merge pull request #37866 from giraffate/correct_default_val…](https://github.com/rails/rails/commit/596e1250cfde2768232432071365a10554517c9c)
- (24bf3f0) [Merge pull request #37882 from kamipo/should_keep_zero_durat…](https://github.com/rails/rails/commit/24bf3f08c096d1cbb934a0265cb707a8d3f9bfc2)
- (d2f53ac) [Switch to the writer DB to generate ASt previews](https://github.com/rails/rails/commit/d2f53ac5c306557c3d3204399b34fccfbd4d6baa)
- (e193acb) [Merge pull request #37666](https://github.com/rails/rails/commit/e193acb118b2bbcb4fa06d727f63a26e62405d9f)
- (e679a4b) [Fix random CI failure due to non-deterministic sorting order](https://github.com/rails/rails/commit/e679a4beb60565e60f6c9eca5e560de4baba6a1d)
- (dde113d) [Fix random CI failure due to non-deterministic sorting order](https://github.com/rails/rails/commit/dde113db74cadbdb80e0b36fedcdf953c95965ec)
- (301edc9) [unlinks Ruby on Rails Tutorial [skip ci]](https://github.com/rails/rails/commit/301edc94f47378e522c199f6313c018e77f30f84)


</details>